workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/InviteMemberModal/InviteMemberModal.tsx

```diff
@@ -0,0 +1,309 @@
+import ClayAlert from '@clayui/alert';
+import ClayButton from '@clayui/button';
+import ClayModal, { useModal } from '@clayui/modal';
+import ClayForm, { ClayInput, ClayCheckbox } from '@clayui/form';
+import { useEffect, useState } from 'react';
+import { Liferay } from '../../liferay/liferay';
+import './inviteMemberModal.scss'
+
+interface InviteMemberModalProps {
+  handleClose: () => void;
+  selectedAccount: Account;
+}
+
+interface CheckboxRole {
+  roleName: string;
+  isChecked: boolean;
+}
+
+export function InviteMemberModal({
+  handleClose,
+  selectedAccount,
+}: InviteMemberModalProps) {
+  const { observer, onClose } = useModal({
+    onClose: () => handleClose(),
+  });
+
+  const [email, setEmail] = useState('');
+  const [firstName, setFirstName] = useState('');
+  const [lastName, setLastName] = useState('');
+  const [checkboxRoles, setCheckboxRoles] = useState<CheckboxRole[]>([]);
+  const [formValid, setFormValid] = useState<boolean>(false);
+
+  const [newUser, setNewUser] = useState<any>();
+  const [accountRoles, setAccountRoles] = useState<any>();
+
+  const listOfRoles = ['Account Administrator', 'App Editor'];
+
+  useEffect(() => {
+    const mapRoles = listOfRoles.map((x) => {
+      return { roleName: x, isChecked: false };
+    });
+    setCheckboxRoles(mapRoles);
+    getAccoutRoles();
+  }, []);
+
+  const jsonBody = () => {
+    return {
+      alternateName: email.replace('@', '-'),
+      emailAddress: email,
+      familyName: lastName,
+      givenName: firstName,
+    };
+  };
+
+  const getAccoutRoles = async () => {
```

Comment:
Please, adjust the function name here.
```suggestion
  const getAccountRoles = async () => {
```